spring-boot-tools/spring-boot-loader-tools/src/main/java/org/springframework/boot/loader/tools/RunProcess.java

```diff
@@ -58,6 +66,7 @@ public int run(boolean waitForProcess, String... args) throws IOException {
 	protected int run(boolean waitForProcess, Collection<String> args)
 			throws IOException {
 		ProcessBuilder builder = new ProcessBuilder(this.command);
+		builder.directory(workingDirectory);
```

Comment:
Actually `null` is valid value. It does nothing but still I feel a bit uncomfortable to just ignore the `null` value. In general I think in similar cases is better to no accept null values at all(if we consifer then as not a valid values) and throw NPE at the constructor. But I guess in this particular case it's enough just to add a note to the javadoc that the `null` for `wokingDirecotry` does not change the working directory.
What do you think?